src/OrbitGl/TimelineTicks.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TIMELINE_IMPL_H_
+#define ORBIT_GL_TIMELINE_IMPL_H_
+
+#include <cstdint>
+#include <set>
+#include <vector>
+
+namespace orbit_gl {
+
+constexpr uint64_t kSecondToNano = 1000 * 1000 * 1000;
```

Comment:
The naming "to" implies an operation, but the operation is not specified. Here, one has to multiply by this value to get from seconds to nanoseconds.

I would prefer `kNanosecondsPerSecond`, `kNanosecondsPerMinute`, `kNanosecondsPerHour`. With this naming, units naturally "cancel out" when you write something like `timestamp_in_hours *  kNanosecondsPerHour`. It's clear that the unit now is nanoseconds.